Automattic/pocket-casts-android
# Introduce Encrypted Logging

## Description


This PR integrates Encrypted Logging feature to the Pocket Casts app. Each Sentry ticket will now contain UUID, that can be later used to browse logs uploaded to the internal service.


## Testing Instructions




1. Change sampling to 1.0 (`PocketCastsCrashLoggingDataProvider#MOBILE_ERROR_SAMPLING`)
1. Install debug app
2. Go to developer settings, trigger a new issue
3. Go to Sentry, find the issue
4. Look for `uuid` section
5. Copy the UUID and use https://mc.a8c.com/encrypted-logs.php . Verify that logs are available.

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
